### DISCUSSION TITLE
Questions about compatibility.

### DISCUSSION THREAD

**Discussion by bingbongboink** [IrisShaders/Iris#615] [Q&A] [answered]
It seems like Iris doesn't work with the Realms service, and I also can't figure out how to add other Fabric mods so that I can use them in conjunction with Iris. Not being able to connect to realms with it is kinda a dealbreaker for me, so I wanna know if it's possible.

**Comment by Ghostly1011** [comment]
Realms don't work with any modified versions of Minecraft due to limitations Mojang has put in place.

**Comment by NoComment1105** [comment, marked as answer]
Using the Iris Installer, if the `Install as fabric mod` checkbox is checked, you are able to install other mods in the `mods` directory and use them with Iris
